src/styles/quarks/_quarks.mixins.scss

```diff
@@ -16,15 +15,16 @@
     font-family: $_name;
     font-style: $_style;
     font-weight: $_weight;
+
     @if $_local {
       src: local($_name), url($_url + '.woff') format('woff'), url($_url + '.woff2') format('woff2');
-    } @else {
+    }
```

Comment:
Dit wringt een beetje in mijn hoofd, maar ik wil er de PR niet voor tegenhouden.

Reply:
Ik ga eens kijken of ik dit (in een andere PR) nog kan aanpassen. 🙂